modules/grpc/otel/tests/test-otel-protobuf-formatter.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2023 Attila Szakacs
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#include <criterion/criterion.h>
```

Comment:
We should include Criterion last as something in the protobuf headers collide with it:

```
In file included from /usr/include/criterion/criterion.h:119,
                 from ../modules/grpc/otel/tests/test-otel-protobuf-formatter.cpp:23:
/usr/include/absl/hash/internal/hash.h:903:27: error: pasting "int_" and "." does not give a valid preprocessing token
  903 |     static std::true_type Test(int);
      |                           ^~~~
```

Reply:
Thanks, fixed I think.